Cargo.toml

```diff
@@ -11,7 +11,7 @@ resolver = "2"
 anyhow = "1"
 async-trait = "0.1"
 # branch is icelake-dev
-apache-avro = { git = "https://github.com/icelake-io/avro.git", rev = "4866a4ad0ed5d6af7160c9b52af898ab6d0551f9", features = ["derive"] }
```

Comment:
Please use `rev` to pin to a version, otherwise downstream project maybe affected when we push changes to branch.